code/src/components/StoryTime/StoryTime.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable max-len */
+/* eslint-disable react/button-has-type */
+import React from 'react';
+import './storytime.css';
+
+export const StoryTime = ({ name, place, animal, horror, food }) => {
+  const Choises = () => {
+    if (animal === 'duck') {
+      animal = 'the duck'
+    } else {
+      animal = 'the frog'
+    }
+    if (horror === 'snakes') {
+      horror = 'pushing them into snake pits'
+    } else if (horror === 'pinapple') {
+      horror = 'adding pinapple on their pizzas'
+    } else {
+      horror = 'throwing them off airplanes'
+    }
+    if (place === 'forest') {
+      place = 'a green forest'
+    } else if (place === 'mountain') {
+      place = 'a stunning mountain valley'
+    } else {
+      place = 'a mystical kingdom'
+    }
+  }
+
+  Choises();
```

Comment:
simple constant to get the choises, used straight after for telling the story, well done